index.html

```diff
@@ -0,0 +1,374 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/0a8239681c.js" crossorigin="anonymous"></script>
+  <title>Smashing Magazine</title>
+</head>
+<body>
+  <header class="d-flex w-100">
+    <div></div>
+    <nav class="w-100 c-gray mija">
+      <ul class="d-flex menu">
+        <li class="logo-section">LOGO</li>
+        <li>Articles</li>
+        <li>Guides</li>
+        <li>Books</li>
+        <li class="medium-dark border-menu padding-menu">Events</li>
+        <li>Membership</li>
+        <li>Jobs</li>
+        <li>Newsletter</li>
+        <li class="darker-menu border-menu padding-menu">More</li>
+        <li class="input-container"><i class="fas fa-search zoom c-black"></i><input type="text" class="menu-input border-menu" placeholder="Search articles..."></li>
+      </ul>
+      
+    </nav>
+  </header>
+
+  <main class="w-100 d-flex v-flex">
+    <section class="sect-1 w-100 c-black mija">
+      <h1 class="in-btn-text c-black">
+        <mark>Don't Miss These Latest Articles</mark>
+      </h1>
+      <div class="top-articles d-flex w-100">
+        <div class="article-1 style-articles">
+          <div class="d-flex ">
+            <img class="img-profile" src="images/profile-image-placeholder.jpg" alt="">
+          <p class="padding-menu">Carrie Webter <cite>wrote</cite></p>
+          </div>
+          <p class="top-article-main-text">Intuitive Design? No Such Thing! <span><i class="fas fa-comment-alt"></i> 	&nbsp; Leave a comment</span></p>
+          
+          <div class="hash d-flex">
+            <p>UI<sup>92</sup></p>
+          <p>Design<sup>457</sup> </p>
+          <p>User Experience<sup>312</sup></p>
+          </div>
+          <i class="fas article-arrow fa-arrow-right"></i>
+        </div>
+        <div class="article-2 style-articles">
+          <div class="d-flex ">
+            <img class="img-profile" src="images/profile-image-placeholder.jpg" alt="">
+          <p class="padding-menu">Saskia Freeke <cite>wrote</cite> </p>
+          </div>
+          <p class="top-article-main-text">Unexpected Learning From Coding Artwork Every Day For Five Years <span><i class="fas fa-comment-alt"></i> 	&nbsp; Leave a comment</span></p>
+          
+          <div class="hash d-flex">
+            <p>Design<sup>457</sup></p>
+            <p>Graphics<sup>356</sup></p>
+          </div>
+          <i class="fas article-arrow fa-arrow-right"></i>
+        </div>
+        <div class="article-3 style-articles">
```

Comment:
This is another article. Kindly wrap __this and all other articles__ inside the `<article>` tag.